.python-version

```diff
@@ -0,0 +1 @@
+2.7.15
```

Comment:
A quick question: wouldn't it force the environment to use exactly `2.7.15`? What would happen if I have, for instance, only `2.7.0` installed?

Reply:
@stukalin Good catch! I'll try to find a solution for that

Reply:
Ah, wait, this is the file to manage the `pyenv`. Weeeellllll, I really don't know if it's really worth it to add a file for a tool which selects a tool to build a tool. I'd personally be against that. You can always use this directive in your working folder and don't commit. Opinions @MatthewDorner @donaldwasserman ?

Reply:
@stukalin That's true, maybe we could add it in the `.gitignore` so that users can set that on their own if needed?

Reply:
This also caused the CI to fail, so I agree it wouldn't be a good idea.

Reply:
so if systems have Python 3 as default, this file alone will not fix it because they still won't have pyenv or the 2.x version of Python? If so then I think I would agree with @stukalin .

Reply:
@MatthewDorner Yep, that seems to be the case here